runtime/autoload/health/provider.vim

```diff
@@ -451,10 +451,11 @@ function! s:check_ruby() abort
 
   let host = provider#ruby#Detect()
   if empty(host)
-    call health#report_warn('Missing "neovim" gem.',
-          \ ['Run in shell: gem install neovim',
-          \  'Is the gem bin directory in $PATH? Check `gem environment`.',
-          \  'If you are using rvm/rbenv/chruby, try "rehashing".'])
+    call health#report_warn('Can''t find `neovim-ruby-host` executable.',
+          \ ['Ensure the `neovim` RubyGem is installed using `gem install neovim`.',
+          \  'Run `gem environment` to ensure the gem bin directory is in $PATH.',
+          \  'If you are using rvm/rbenv/chruby, try "rehashing".',
+          \  'See `:help g:ruby_host_prog` for non-standard gem installations.'])
```

Comment:
health.vim automatically highlights `:help foo` as navigable help tags. The backticks mess it up though, so I removed them.